update_actions.sh

```diff
@@ -0,0 +1,42 @@
+
+#!/bin/bash
+set -e
+
+VERSION=$1
+
+if [ -z "$VERSION" ]; then
+        echo "missing version"
+        exit 1
+fi
+
+actionsRepos=(delete-namespace
+              build
+              destroy-preview
+              deploy-preview
+              deploy-stack
+              namespace
+              pipeline
+              push
+              create-namespace
+              destroy-pipeline
+              login
+              destroy-stack
+              apply)
+              
+for repo in "${actionsRepos[@]}"
+do
+    echo "$repo"
+    git clone --depth 1 git@github.com:okteto/$repo.git
+    pushd $repo
+    git config user.name "okteto"
+    git config user.email "ci@okteto.com"
+    sed -iE 's_FROM\ okteto\/okteto\:latest_FROM\ okteto\/okteto\:'$VERSION'_' Dockerfile
```

Comment:
make sure this idempotent, and things still work in the job is executed twice for the same `$VERSION`